examples/with-eth-passkeys-galore/src/pages/index.tsx

```diff
@@ -67,6 +67,10 @@ export default function Home() {
   };
 
   const signMessageViem = async (data: signingFormData) => {
+    if (passkeyClient && passkeyClient.config && wallet) {
```

Comment:
we're going to need this for other methods like `signTransactionViem` and `signTransactionEthers` as well. however, in order to avoid repeating ourselves, the easier solution IMO is to do this at the end of the `createSubOrgAndWallet` call right here: https://github.com/tkhq/sdk/blob/402c3bd55a07160ff55e9d53a3ec49aaf5118c89/examples/with-eth-passkeys-galore/src/pages/index.tsx#L141-L142

also, from a syntax perspective, you can use something like `passkeyClient?.config.organizationId = wallet.subOrgId;`

this implicitly does the `if (passkeyClient && ...`) check under the hood